composer.json

```diff
@@ -16,16 +16,33 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.9",
+        "symfony/console": "~3.3|~4.0",
+        "symfony/config": "~3.3|~4.0",
+        "symfony/dependency-injection": "~3.3|~4.0",
+        "symfony/framework-bundle": "~3.3|~4.0",
+        "symfony/http-kernel": "~3.3|~4.0",
+        "symfony/polyfill-php70": "~1.0",
         "symfony/security-acl": "~2.8"
```

Comment:
The problem here is that 2.7 is the only one available, idk where the rest comes from, but they are not from the `symfony/security-acl` repo.
```
versions : dev-master, 3.0.x-dev, v3.0.0, 2.8.x-dev, v2.8.0, 2.7.x-dev, v2.7.33, v2.7.32, v2.7.31, v2.7.30, v2.7.29, v2.7.28, v2.7.27, v2.7.26, v2.7.25, v2.7.24, v2.7.23, v2.7.22, v2.7.21, v2.7.20, v2.7.19, v2.7.18, v2.7.17, v2.7.16, v2.7.15, v2.7.14, v2.7.13, v2.7.12, v2.7.11, v2.7.10, v2.7.9, v2.7.8, v2.7.7, v2.7.6, v2.7.5, v2.7.4, v2.7.3, v2.7.2, v2.7.1, v2.7.0, v2.7.0-BETA2, v2.7.0-BETA1, 2.6.x-dev, v2.6.13, v2.6.12, v2.6.11, v2.6.10, v2.6.9, v2.6.8, v2.6.7, v2.6.6, v2.6.5, v2.6.4, v2.6.3, v2.6.2, v2.6.1, v2.6.0, v2.6.0-BETA2, v2.6.0-BETA1, 2.5.x-dev, v2.5.12, v2.5.11, v2.5.10, v2.5.9, v2.5.8, v2.5.7, v2.5.6, v2.5.5, v2.5.4, v2.5.3, v2.5.2, v2.5.1, v2.5.0, v2.5.0-RC1, v2.5.0-BETA2, v2.5.0-BETA1, 2.4.x-dev, v2.4.10, v2.4.9, v2.4.8, v2.4.7, v2.4.6, v2.4.5, v2.4.4, v2.4.3, v2.4.2, v2.4.1, v2.4.0, v2.4.0-RC1, v2.4.0-BETA2, v2.4.0-BETA1
```